redbot/cogs/mod/names.py

```diff
@@ -69,6 +69,105 @@ class ModInfo(MixinMeta):
+        s_acts = [c for c in user.activities if c.type == discord.ActivityType.streaming]
+        s_act = s_acts[0] if s_acts else None
+        act = (
+            _("Streaming on {platform}: [{name}{sep}{game}]({url})").format(
+                platform=s_act.platform,
+                name=discord.utils.escape_markdown(s_act.name),
+                sep=" | " if s_act.game else "",
+                game=discord.utils.escape_markdown(s_act.game) if s_act.game else "",
+                url=s_act.url,
+            )
+            if s_act and s_act.name and hasattr(s_act, "game")
+            else discord.utils.escape_markdown(s_act.name)
+            if s_act and s_act.name
+            else None
+        )
+        return act, discord.ActivityType.streaming
+
+    def handle_listening(self, user):
+        l_acts = [c for c in user.activities if c.type == discord.ActivityType.listening]
+        l_act = l_acts[0] if l_acts else None
+        act = (
+            _("Listening: [{title}{sep}{artist}]({url})").format(
+                title=l_act.title,
+                sep=" | " if l_act.artists[0] else "",
+                artist=discord.utils.escape_markdown(l_act.artist) if l_act.artist else "",
+                url=f"https://open.spotify.com/track/{l_act.track_id}",
+            )
+            if l_act and hasattr(l_act, "title")
+            else l_act.name
+            if l_act and l_act.name
+            else None
+        )
+        return act, discord.ActivityType.listening
+
+    def handle_watching(self, user):
+        w_acts = [c for c in user.activities if c.type == discord.ActivityType.watching]
+        w_act = w_acts[0] if w_acts else None
+        act = (
+            _("Watching: {name}").format(name=w_act.name if w_act else None)
+            if w_act and hasattr(w_act, "name")
+            else w_act.name
+            if w_act and w_act.name
+            else None
+        )
+        return act, discord.ActivityType.watching
+
+    def get_status_string(self, user):
+        string = ""
+        for a in [
+            self.handle_custom(user),
+            self.handle_playing(user),
+            self.handle_listening(user),
+            self.handle_streaming(user),
+            self.handle_watching(user),
+        ]:
+            status_string, status_type = a
+            if status_string is None:
+                continue
+            if status_type == discord.ActivityType.custom:
+                string += "{}\n".format(status_string)
+            elif status_type == discord.ActivityType.playing:
+                string += "{}\n".format(status_string)
+            elif status_type == discord.ActivityType.streaming:
+                string += "{}\n".format(status_string)
+            elif status_type == discord.ActivityType.listening:
+                string += _("{}\n".format(status_string))
+            elif status_type == discord.ActivityType.watching:
+                string += "{}\n".format(status_string)
```

Comment:
This can be simplified to
```suggestion
            string += "{}\n".format(status_string)
```